experimental/FTheoryTools/src/auxiliary.jl

```diff
@@ -198,12 +198,13 @@ function _kodaira_type(id::MPolyIdeal{<:MPolyRingElem}, ords::Tuple{Int64, Int64
     # five times and see if we get agreement among all of the results
     num_gens = ngens(parent(f))
     gauge2s, f2s, g2s, d2s = [], [], [], []
+    rng = MersenneTwister()
```

Comment:
@benlorenz This is the first thing which @apturner and I could think of. But not sure if this is the best choice. Maybe you have thoughts?

In any case, this should be one of the key changes you were asking for.

Reply:
Thank you. Just adjusted accordingly.